specification/app/resource-manager/Microsoft.App/preview/2022-01-01-preview/EasyAuthConfigs.json

```diff
@@ -408,6 +408,102 @@
         "azureStaticWebApp": {
           "$ref": "#/definitions/AzureStaticWebApp",
           "description": "The configuration settings of the Azure Static Web Apps provider."
+        },
+        "customOpenIdConnectProviders": {
```

Comment:
May you add this property in the example file if possible?

Reply:
Synced with @erich-wang offline and he would like to get this PR merged firstly to catch up the main branch merge then will add example in other PRs later.